Index: src/CMakeLists.txt
===================================================================
--- src.orig/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -23,7 +23,7 @@ add_dune_tbb_flags(dunecommon)
 # add subdirectories to execute CMakeLists.txt there
 add_subdirectory(bin)
 add_subdirectory(cmake)
-add_subdirectory(doc)
+#add_subdirectory(doc)
 add_subdirectory(dune)
 add_subdirectory(lib)
 add_subdirectory(share)
